### DISCUSSION TITLE
Gavish-Graves Formulation

### DISCUSSION THREAD

**Discussion by TiesWestendorp** [hakank/hakank#14] [General]
Came across this repo through a Google search, and I would like to point out that though the model formulation was well found - it is by no means novel. This is in fact exactly the Gavish-Graves formulation for TSP.

https://github.com/hakank/hakank/blob/0145ada7c14b996c4c57ed8010941911c40edfb7/minizinc/tsp.mzn#L78-L96

**Comment by hakank** [comment]
Hi Ties.

Thanks for the comment. 

Just to make it straight, the text is a quote from the GLPK:s example tsp.mod:
```
TSP, Traveling Salesman Problem

Written in GNU MathProg by Andrew Makhorin <mao@mai2.rcnet.ru> 
```

Perhaps I should have made that more clear.  I've now added some clarification (c4aebc0).
